### DISCUSSION TITLE
Pass variables from template to js file.

### DISCUSSION THREAD

**Discussion by nul77** [breezefront/community#77] [1. Help]
How can I pass variables from a template to an external script? The script is registered in referenceBlock breeze.js in breeze_default.xml.
The component is initialised via text/x-magento-init script like this:
 `<script type="text/x-magento-init">
            {
                "*": {
                    "FME_GiftWrapper/js/modal": {
                        "symbol": "<?= $escaper->escapeJs($block->getCurrentCurrencySymbol()) ?>"
                    }
                }
            }
        </script>`
        
How can I use the value for the 'symbol' variable in my js-file?
        
Thanks in advance, Tom

**Comment by vovayatsyuk** [comment]
It should be the same as you had in the Luma-based script.

If it's a widget - `this.options.symbol`, or if it's a view (uiComponent) - `this.symbol`

**Comment by nul77** [comment]
Thank you for your reply.
There is no external Luma-based script. Just a big inline script I'm attempting to replace.
